app/views/disassociate_ucr.scala.html

```diff
@@ -15,45 +15,55 @@
  *@
 
 @import forms.DisassociateUcr
-@import forms.MucrOptions
-@import views.components.fields.RadioOption
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import config.AppConfig
 @import views.Title
 @import forms.DisassociateKind
 
-@this(main_template: views.html.main_template)
+@this(govukLayout: views.html.gds_main_template,
+        govukButton: GovukButton,
+        govukRadios: GovukRadios,
+        exportsInputText: components.gds.exportsInputText,
+        errorSummary: components.errorSummary,
+        formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF,
+        appConfig: AppConfig)
```

Comment:
I know it's a pattern and consistent with rest pages, but appConfig is not used here and can be removed. It'll make testing easier

Reply:
👍